notebooks/example.ipynb

```diff
@@ -182,24 +139,163 @@
+    "Let's explore using the `config.compare()` method to compare two two config files."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71d27692",
+   "metadata": {},
+   "source": [
+    "We again need to start by importing the `uwtools.api.config` from the uwtools python package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7fb40514",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uwtools.api import config\n",
+    "from uwtools.api.logging import use_uwtools_logger\n",
+    "use_uwtools_logger()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0103f44",
+   "metadata": {},
+   "source": [
+    "Please review the [config.compare() documentation](https://uwtools.readthedocs.io/en/main/sections/user_guide/api/config.html#uwtools.api.config.compare) for full information on this function's arguments."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b121dc29",
+   "metadata": {},
+   "source": [
+    "For example, let's compare two Fortran namelist files with differences:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8279b5c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "cat ./fixtures/config_test_file_a.nml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e3232247",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "cat ./fixtures/config_test_file_b.nml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5dcff125",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# path of nml file 'a' shown above\n",
+    "file_a = \"./fixtures/config_test_file_a.nml\"\n",
+    "\n",
+    "# path of nml file 'b' shown above\n",
+    "different_file_b = \"./fixtures/config_test_file_b.nml\"\n",
+    "\n",
+    "config.compare(file_a, different_file_b)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c16c38d0",
+   "metadata": {},
+   "source": [
+    "The `config()` method returns `False` to denote the files are different. The UW logger shows the difference, of one file containing `n = 88`, and one file containing `n = 99`.\n",
+    "\n",
+    "Now to compare two semantically equivalent files:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "370149a3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "cat ./fixtures/config_test_file_a.nml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bd444fba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "cat ./fixtures/config_test_file_c.nml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9d658b72",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# path of nml file that contains 'greeting = \"Hello\" recipient = \"World\"'\n",
```

Comment:
When all we have is source code (i.e. when we don't have notebooks), we must use code comments for commentary because we have no other choice. But code comments are distracting, and a selling point for notebooks is their ability to separate commentary and code into separate cells for better readability / storytelling. So I feel like we should have few to no code comments in these notbooks: Whatever we want to say, we can (and I think probably should) say in commentary, rather than code, cells. I'm sure there are edge cases where code comments might still be useful, but I think it would be good to leverage notebooks for what they're good at and keep the commentary and code separate. I realize that previous sections might also need an update along these lines.

Reply:
Removed the comments, these are also not necessary with the bash cell magic added.